algosdk/v2client/models/simulate_request.py

```diff
@@ -16,11 +16,36 @@ def dictify(self) -> Dict[str, Any]:
         return {"txns": [txn.dictify() for txn in self.txns]}
 
 
+class SimulateTraceConfig(object):
```

Comment:
```suggestion
class SimulateTraceConfig:
```

Reply:
right, I can move that later, just mucking around and warming up with SDKs.